db/migrate/20200304203650_create_organism_age_facet_and_backfill_data.rb

```diff
@@ -0,0 +1,21 @@
+class CreateOrganismAgeFacetAndBackfillData < Mongoid::Migration
+  def self.up
+    SearchFacet.where(identifier: 'organism_age').destroy
+    bq_dataset = SearchFacet.big_query_dataset
+    update_command = "UPDATE #{CellMetadatum::BIGQUERY_TABLE} "
+    update_command += "SET organism_age__seconds = CAST((organism_age * #{SearchFacet::TIME_MULTIPLIERS['years']}) AS NUMERIC) "
```

Comment:
how about adding a "AND organism_age__unit_label = 'year'" to this where clause just to be on the safe side?

Reply:
4c3b606